src/App.tsx

```diff
@@ -1,9 +1,33 @@
-import "./App.css";
-import * as React from "react";
-import Container from "./components/Container";
+/* eslint-disable react/function-component-definition */
+import { FC, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// selectors
+import { coordinatesSelector } from 'store/location-service/selectors';
+// functions
+import { getWeathers, getWeathersInfoByCity } from 'store/weather-service/action';
+// types
+import { Coord } from 'core/types';
+// components
+import Content from 'components/Content';
+import Sidebar from 'components/Sidebar';
+// styles
+import s from './App.module.scss';
 
-function App() {
-  return <Container />;
-}
+const App: FC = () => {
+	const dispatch = useDispatch();
+	const coordinates: Coord = useSelector(coordinatesSelector);
+
+	useEffect(() => {
```

Comment:
in my opinion that actions will be better used inside each component. if u need getWeathers only in the sidebar - use it sidebar

Reply:
done